code/modules/antagonists/abductor/equipment/glands/mindshock.dm

```diff
@@ -15,6 +15,10 @@
 	for(var/mob/living/carbon/H in orange(4,T))
```

Comment:
if we use the variable, we should rename bad vars while we're here. change H (and places that use it) to something better!